wordpress-mobile/WordPress-iOS
# [Gutenberg] Unsupported fallback tracks

This PR replaces https://github.com/wordpress-mobile/WordPress-iOS/pull/14353
Fixes parts of https://github.com/wordpress-mobile/gutenberg-mobile/issues/2358

Implements changes from: https://github.com/WordPress/gutenberg/pull/23450
`gutenberg-mobile` ref: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2449

We are adding:
- `gutenberg_unsupported_block_webview_shown`
  - params: `"block": "{block_internal_name}"`
- `gutenberg_unsupported_block_webview_closed` 
  - params: `"action": "save" / "dismiss"` accordingly.


cc @marecar3 

To test:

- Run Metro server on https://github.com/wordpress-mobile/gutenberg-mobile/pull/2449  to activate the webview option on unsupported blocks ensuring that the required JS code is running.
- Open the unsupported block web view
  - Check that `gutenberg_unsupported_block_webview_shown` is triggered.
- Close the web view by dismissing and by saving
  - Check that `gutenberg_unsupported_block_webview_closed` is triggered with the corresponding property.

PR submission checklist:
